challenge.rb

```diff
@@ -45,9 +53,17 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if ac_working && current_temp > desired_temp
+    puts "turn on the AC"
+  elsif ac_working && current_temp <= desired_temp
+    puts "just right"
+  elsif current_temp > desired_temp
+    puts "fix AC now! too hot!!"
+  else
+    puts "fix the AC whenever you have the chance. It's cool"
+  end
```

Comment:
Good use of logic. Again, though, this method should be returning a value rather than printing one.